### DISCUSSION TITLE
What should  I do?????

### DISCUSSION THREAD

**Discussion by wjwpli** [EssentialsX/Essentials#4990] [Q&A] [answered]
I want normal players have the access to use quickshop(create chestshop and default the shop they created before but)
which permission group should they be added into?

**Comment by Chew** [comment, marked as answer]
I can't say I understand, none of what you asked applies to or exists in EssentialsX

**Comment by wjwpli** [comment]
sorry bro，I mistake this with another one，I will delete the way later.



| |
***@***.***
|
|
***@***.***
|




---- Replied Message ----
| From | ***@***.***> |
| Date | 07/13/2022 08:02 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [EssentialsX/Essentials] What should I do????? (Discussion #4990) |

I can't say I understand, none of what you asked applies to or exists in EssentialsX
